snaps_k8s/playbooks/k8/weave_conf_deletion.yaml

```diff
@@ -54,7 +54,7 @@
        {
          "name": "minion-cni-network",
          "type": "multus",
-         "kubeconfig": "{{ KUBERNETES_PATH }}/node-kubeconfig.yaml",
+         "kubeconfig": "/etc/kubernetes/kubelet.conf",
```

Comment:
Still don't like the harcoding of /etc/kubernetes here

Reply:
incorported

Reply:
Thanks!